### DISCUSSION TITLE
New script which restores the "Never update Firefox" option

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [Aris-t2/CustomJSforFx#87] [Ideas]
I don't even know why the Mozilla removed it, but it was a good option for update, e.g. if you never wanted to update the Firefox.

**Comment by Aris-t2** [comment]
This is beyond what scripts are doing here. Users who want to block updates should not install Mozillas background service and enable the policy to block updates in registry.

https://winaero.com/how-to-disable-firefox-background-updates/

**Comment by Aleksandra0056** [comment]
I'm closing this as not relevant.
